src/main/java/view/context_menu/listeners/ContextMenuActionListener.java

```diff
@@ -1,53 +1,46 @@
 package view.context_menu.listeners;
 
 import burp.*;
-import models.vulnerability.Evidence;
-import services.TemplateService;
+import models.evidences.EvidenceArchive;
+import models.tabs_manager.TabsManager;
 import utilities.Util;
-import view.new_vulnerability.NewVulnerabilityTab;
+import view.issues_tab.NewIssueTab;
 
 import javax.swing.*;
 import java.awt.*;
 import java.awt.event.ActionEvent;
 import java.awt.event.ActionListener;
 import java.io.File;
-import java.io.IOException;
-import java.nio.file.Files;
-import java.nio.file.Path;
 
 public class ContextMenuActionListener implements ActionListener {
 
     private IBurpExtenderCallbacks callbacks;
     private IExtensionHelpers helpers;
     private IContextMenuInvocation invocation;
     private Util util;
-    private final NewVulnerabilityTab newVulnerabilityTab;
-    String requestHeader = "/* \n * REQUEST \n */\n\n";
-    String responseHeader = "/* \n * RESPONSE \n */\n\n";
+    private final TabsManager tabsManager;
 
 
-    public ContextMenuActionListener(final IBurpExtenderCallbacks callbacks, final IExtensionHelpers helpers, NewVulnerabilityTab newVulnerabilityTab, IContextMenuInvocation invocation ) {
+    public ContextMenuActionListener(final IBurpExtenderCallbacks callbacks, final IExtensionHelpers helpers, final TabsManager tabsManager, IContextMenuInvocation invocation) {
         this.callbacks = callbacks;
         this.helpers = helpers;
         this.util = new Util(this.callbacks);
         this.invocation = invocation;
-        this.newVulnerabilityTab = newVulnerabilityTab;
+        this.tabsManager = tabsManager;
     }
 
     @Override
     public void actionPerformed(ActionEvent e) {
-        switch (e.getActionCommand()) {
-            case "as new vulnerability/notification" -> {
-                this.defineCampsNewVulnerabilityTab(invocation.getSelectedMessages()[0]);
-                this.selectAppSecFlowTab();
-            }
-            case "as evidence" -> this.createEvidencesInTempFiles(invocation.getSelectedMessages().length);
-            case "as new vulnerability w. evidence" -> {
-                this.createEvidencesInTempFiles(invocation.getSelectedMessages().length - 1);
-                this.defineCampsNewVulnerabilityTab(invocation.getSelectedMessages()[invocation.getSelectedMessages().length-1]);
-                this.selectAppSecFlowTab();
-            }
-        }
+//        switch (e.getActionCommand()) {
```

Comment:
Should we remove commented code?

Reply:
This one, i would like to maintain, because of a feature in the future. But will be removed soon.